lib/parameter_substitution/expression.rb

```diff
@@ -77,6 +91,28 @@ def validate_expressions
       end
     end
 
+    def unknown_method_messages
+      unknown_parameter_methods.map_compact do |param, methods|
+        method_string = "method#{methods.size > 1 ? 's' : ''}"
```

Comment:
We already have `active_support` as a requirement in this project, so you could `require 'active_support/inflector'` and then use `String#pluralize`. 
https://apidock.com/rails/String/pluralize

Reply:
Though this is an elegant solution 👍 .

If you decide to keep your own solution, it's probably worth DRYing it up into a method since it's repeated on line `106`. 

Edit: I see this was already the pattern in place in this file. Up to you whether you want to use `ActiveSupport` or not.

Reply:
Well I noticed that active_support was added a development dependency not a runtime dependency which means it not really available at runtime so I avoided using pluralize.

Note:
Since we only use this in `web` which has `active_support'` it all works in the end but strictly speaking `pluralize` should not be used in the project.

Reply:
I dried up pluralizing bit and pushed it up.

Reply:
>I noticed that active_support was added a development dependency not a runtime dependency

Good catch. I missed that.